### DISCUSSION TITLE
computer vision 101

### DISCUSSION THREAD

**Discussion by TinyHust** [TinyHust/notes#440] [AI - ML - DL - CS - DS]
## Tools
- [opencv](https://github.com/opencv/opencv)
- [YOLO](https://docs.ultralytics.com/)
- [easyocr](https://www.jaided.ai/easyocr/documentation/)
- [roboflow](https://roboflow.com/)

---
- [NVIDIA Container Toolkit](https://github.com/NVIDIA/nvidia-container-toolkit)

**Comment by mediumhust** [comment]
## YOLO

### Examples
- [Real-Time Class-Wise Vehicle Counting and Tracking](https://github.com/AarohiSingla/Vehicle-detection-and-tracking-classwise-using-YOLO11)
- [Object-Detection-Web-Application-with-Flask-and-YOLOv9](https://github.com/AarohiSingla/Object-Detection-Web-Application-with-Flask-and-YOLOv9)
- [train model](https://github.com/AarohiSingla/YOLO11-Custom-Object-Detection)
- [train dataset](https://github.com/AarohiSingla/YOLO11-Instance-Segmentation)

### Roboflow
- Real-Time Stream Processing
  - https://www.youtube.com/watch?v=hAWpsIuem10
  - https://github.com/roboflow/supervision/tree/develop/examples/time_in_zone

**Comment by mediumhust** [comment]
## Libs
- [mediapipe](https://github.com/google-ai-edge/mediapipe)

**Comment by mediumhust** [comment]
## Examples
- [camera geasture](https://github.com/wonderwhy-er/camera-geasture-drawing)

**Comment by mediumhust** [comment]
## Term
- ByteTrack
